internal/db/schema/migrations/postgres/5/01_oidc.up.sql

```diff
@@ -0,0 +1,85 @@
+begin;
```

Comment:
Should we have waited to merge `/4/` until this PR was in since the user wouldn't have been able to show Email or FullName before this anyways?

Reply:
It's a good question.  Whether or not Email/FullName showed would depend on the Provider implementation.  For Okta and Auth0 they wouldn't have shown, but you could have implemented Dex as a broker and returned whatever you'd like for claims with just the default `openid` scope.